console/lockable_trait.rst

```diff
@@ -30,7 +30,7 @@ that adds two convenient methods to lock and release commands::
             }
 
             // If you prefer to wait until the lock is released, use this:
-            // $this->lock(true);
+            // $this->lock('lockname', true);
```

Comment:
```null``` could do the trick as any string. Using ```null``` result in using the name define for the command. My point is to use the blocking functionnality of the trait, we need to add a first parameter before ```true```.